<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>543</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>9000</width>
    <height>90000</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QFrame" name="Frame">
      <property name="styleSheet">
       <string notr="true">QFrame {
    background-color: #41b6e6;  /* Primary Blue Background */
    border: 2px solid #ccc;
    padding: 10px;
    border-radius: 5px;
}
        QPushButton {
            background-color: #db3eb1;  /* Neon Pink */
            border: none;
            color: white;
            padding: 10px;
            text-align: center;
            text-decoration: none;
            font-size: 16px;
            margin: 4px 2px;
            cursor: pointer;
            border-radius: 8px;
        }
        QPushButton:hover {
            background-color: #db3eb1;  /* Neon Pink */
        }
        QTextEdit, QLineEdit {
            border: 1px solid #41b6e6;  /* Light Blue Border */
            background-color: #e6f7ff;  /* Light visible color */
            color: #000000;  /* Black text for better readability */
        }</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="frameLayout">
       <item>
        <widget class="QLabel" name="Title">
         <property name="font">
          <font>
           <family>Stencil</family>
           <pointsize>28</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
        color: #000000;  /* Black */
        text-align: center;
        background-color: transparent;
}</string>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Plain</enum>
         </property>
         <property name="text">
          <string>NovaMind</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Input Math Equation:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="buttonLayout">
         <item>
          <widget class="QToolButton" name="toolButton_Arithmetic">
           <property name="styleSheet">
            <string notr="true">QToolButton {
    background-color: #db3eb1;  /* Neon Pink */
    color: #ffffff;  /* White text */
    border: none;
    border-radius: 5px;
    padding: 5px;
}</string>
           </property>
           <property name="text">
            <string>Arithmetic</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_Algebra">
           <property name="styleSheet">
            <string notr="true">QToolButton {
    background-color: #db3eb1;  /* Neon Pink */
    color: #ffffff;  /* White text */
    border: none;
    border-radius: 5px;
    padding: 5px;
}</string>
           </property>
           <property name="text">
            <string>Algebra</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_Trigonometry">
           <property name="styleSheet">
            <string notr="true">QToolButton {
    background-color: #db3eb1;  /* Neon Pink */
    color: #ffffff;  /* White text */
    border: none;
    border-radius: 5px;
    padding: 5px;
}</string>
           </property>
           <property name="text">
            <string>Trigonometry</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_Calculus">
           <property name="styleSheet">
            <string notr="true">QToolButton {
    background-color: #db3eb1;  /* Neon Pink */
    color: #ffffff;  /* White text */
    border: none;
    border-radius: 5px;
    padding: 5px;
}</string>
           </property>
           <property name="text">
            <string>Calculus</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_Graph">
           <property name="styleSheet">
            <string notr="true">QToolButton {
    background-color: #db3eb1;  /* Neon Pink */
    color: #ffffff;  /* White text */
    border: none;
    border-radius: 5px;
    padding: 5px;
}</string>
           </property>
           <property name="text">
            <string>Graph</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_Symbols">
           <property name="styleSheet">
            <string notr="true">QToolButton {
    background-color: #db3eb1;  /* Neon Pink */
    color: #ffffff;  /* White text */
    border: none;
    border-radius: 5px;
    padding: 5px;
}</string>
           </property>
           <property name="text">
            <string>Symbols</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="OutputFrame">
         <property name="styleSheet">
          <string notr="true">QFrame {
    border: 1px solid #41b6e6;  /* Light Blue Border for Output Labels */
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="EquationEdit">
         <property name="maximumSize">
          <size>
           <width>2200</width>
           <height>100</height>
          </size>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Sunken</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="SubmitButton">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    background-color: #db3eb1;  /* Neon Pink */
    color: #fffff1;  /* White */
    border: 1px solid #ffffff;  /* White Border */
    border-radius: 10px;
    padding: 6px 10px;
    font-size: 12px;
    font-weight: bold;
}
        QPushButton:hover {
            background-color: #db3eb1;  /* Neon Pink */
        }</string>
         </property>
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ResultLabel">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Solution:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ResultImageLabel">
         <property name="minimumSize">
          <size>
           <width>125</width>
           <height>70</height>
          </size>
         </property>
         <property name="text">
          <string>Graph:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>714</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNovaMind_Form">
    <property name="title">
     <string>NovaMind Form</string>
    </property>
   </widget>
   <addaction name="menuNovaMind_Form"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
